astrocut/tests/test_make_cube.py

```diff
@@ -48,6 +49,56 @@ def test_make_cube(tmpdir):
     hdu.close()
     
 
+def test_update_cube(tmpdir):
+    """
+    Testing the make cube and update cube functionality for TICACubeFactory by making a bunch of test FFIs, 
+    making the cube with first half of the FFIs, updating the same cube with the second half,
+    and checking the results.
+    """
+
+    cube_maker = TicaCubeFactory()
+
+    img_sz = 10
+    num_im = 100
+    
+    ffi_files = create_test_ffis(img_sz, num_im, product='tica', dir_name=tmpdir)
+
+    cube_file = path.join(os.getcwd(), "out_dir", "test_update_cube.fits")
+
+    cube_maker.make_cube(ffi_files[0:num_im//2], cube_file, verbose=False)
```

Comment:
since this is our test of the make_cube of TICA, can you add some assertions (like we have for the SPOC cubes above)?

if it makes more sense to split out into a separate test function that's fine too

Reply:
you call `make_cube()` with no assertions until after you call `update_cube()`

since these are two separate functions, i would prefer to have some assertions on _just_ the `make_cube()` portion.

Reply:
Just added some assertions.

Since it's repetitive code, I considered moving them to a separate function and calling that function in for both the `make_cube` and `update_cube` `cube_file`s, but I'm not sure how `pytest` would react to that so it'll need some testing. Let me know if you think it's worth it or if you think it should be left as is. 

As for separating the `make_cube` portion from `update_cube`, I didn't see what there was to gain in doing that, especially since `update_cube` relies on the output of `make_cube`, so it seemed logical to keep them together.

Reply:
yes, thank you, this is exactly what i was looking for